client/my-sites/plans/modernized-layout.tsx

```diff
@@ -0,0 +1,150 @@
+/**
+ * - section styles borrowed from /my-sites/stats, adapted to modernize the header and navbar.
+ * - .navigation styles borrowed from /blocks/stats-navigation, adapted to modernize the plans navigation component.
+ *  This is a temporary solution until we can use a Layout component.
+ */
+
+import { css, Global } from '@emotion/react';
+import { memo } from '@wordpress/element';
+
+const ModernizedLayout: React.FunctionComponent< {
+	section: string;
+	subSection?: string;
+	dropShadowOnHeader?: boolean;
+} > = ( { section, subSection, dropShadowOnHeader = true } ) => {
+	const backgroundColor = '#fdfdfd';
+	const sectionMaxWidth = '1224px';
+	const layoutContentPaddingTop = '79px';
+	const sidebarAppearanceBreakPoint = '783px';
+	// from @automattic/components/src/highlight-cards/variables.scss:
+	const customMobileBreakpoint = '660px';
+	const verticalMargin = '32px';
+
+	const globalOverrides = css`
+		.${ subSection ?? section }__section-header {
```

Comment:
These variable class names are (ought to be) really temp stuff. Ideally, what's mapped on the front e.g. `.plans__section-header` should be searchable in the code base.

Reply:
refactored back to static classes for now